contracts/FlowIDTableStaking.cdc

```diff
@@ -404,30 +395,31 @@ pub contract FlowIDTableStaking {
             self.tokensRequestedToUnstake = delegatorRecord.tokensRequestedToUnstake
         }
 
-        pub fun totalTokensInRecord(): UFix64 {
+        access(all) view fun totalTokensInRecord(): UFix64 {
             return self.tokensStaked + self.tokensCommitted + self.tokensUnstaking + self.tokensUnstaked + self.tokensRewarded
         }
     }
 
-    pub resource interface NodeStakerPublic {
-        pub let id: String
+    access(all) resource interface NodeStakerPublic {
+        access(all) let id: String
     }
 
+    access(all) entitlement NodeOperator
```

Comment:
It just means that the type definition is accessible to the public. If it wasn't `access(all)`, then you couldn't use it to create an entitled capability in a transaction because it wouldn't be accessible to the transaction